foundations/config.yaml

```diff
@@ -6,11 +6,14 @@ metadata:
 data:
   foundations.yaml: |
     foundations:
+    #  clusterCode: "bocchi"
+    #  clusterAddress: "https://k8s.bocchi.rocks:4443"
```

Comment:
Nah K8s should be autogenerated, because there will be more stuff on intranet domain. Also 4443 looks wrong actually, why do we do that?

Reply:
> why do we do that?
We do that because when we give a kubeconfig to a user they need the right clusterAddress. The code is a bit less relevant but it's picked up by tools such as shell prompts.

> Nah K8s should be autogenerated
I honestly have no idea how to do that except run more scripts in bootstrap.sh that queries the control plane and then `sed`s these values. Though I do think that's a decent plan.

Reply:
Sorry I wasn't clear. I meant the domain shouldn't be set for each individual component. But rather just K8s.{intranet domain}

Reply:
renamed them again to `clusterName` and `intranetDomain` respectively